RELEASE.rst

```diff
@@ -1,6 +1,20 @@
 Release Notes
 -------------
 
+Version 0.2.0
+=============
+
+- Implemented authorization for taxonomy pieces of REST API.
+- Styled full text search box per the designer.
+- Updated REST API docs.
+- Imported static assets during course import.
+- Added abstract base model to contain audit fields.
+- Added faceted search for taxonomy items.
+- Added Additional APIs for roles.
+- Switched to using get_perms for cleaner code.
+- Added JavaScript infrastructure to run unit tests.
+- Added slide out panel for learning resource details.
+- Styled left hand navigation pane with glyphs.
```

Comment:
Consider replacing the bulleted list with this:

Version 0.2.0
=============

- The search bar performs full-text search over the learning resources in 
  the repository, the search results replace the contents of the listing page. 

- Full-text search includes taxonomy facets.

- Learning resources details are displayed in a panel that slides out from 
  the right side of the page.

- Glyphs for learning resources types are displayed in the left side panel for 
  facets.

- LORE's RESTful web service documentation is available. 
  (http://docs.lore.apiary.io)
 
- Authorizations are in place for taxonomy endpoints in LORE's web service.

- Relationships between learning resources and static assets are captured.

- Roles app has additional features. 

Other Changes
*************

- Switched to using get_perms for cleaner code.
- Added JavaScript infrastructure to run unit tests.